### ISSUE TITLE
Minigui: listView and treeView seem to be missing

### ISSUE THREAD

**Issue by Menelion** [adamdruppe/arsd#144]
Hi Adam,  
I believe, Minigui lacks listView and treeView, am I wrong? I'm especially interested in Win32 versions :).  
My D knowledge is not too good to add them myself: maybe I'll help you in this some time when I learn the stuff better, but I at least decided to log it for now.  
Thanks!

**Comment by adamdruppe** [comment]
Right, I haven't implemented them yet. I think they're easy on Windows though, just create the window (just like button for example), set some styles, and then do some api to add items.

Of course, the Linux one will be a bit trickier. But can do that later. Also I kinda want to do a fancy data binding thing but that can come later too...

**Comment by adamdruppe** [comment]
there are so so so many features and details available on the Windows controls that allowing it all would take eons. But I can do basic functionality fairly quickly if you need it now. Or if you want to try to do yourself, it can wait too :)
